doc/ref_model/chapters/chapter02.rst

```diff
@@ -0,0 +1,366 @@
+Very Low Latency - Edge Sites                    very-low-latency-edge   ✅                           ✅                                      Labels a host/node as located in an edge site, for workloads requiring low latency (specify value) to final users or geographical distribution.                
+Ultra Low Latency - Edge Sites                   ultra-low-latency-edge  ✅                           ✅                                      Labels a host/node as located in an edge site, for workloads requiring low latency (specify value) to final users or geographical distribution.                
+Fixed function accelerator                       compute-ffa             ❌                           ✅                                      Labels a host/node that includes a consumable fixed function accelerator (non-programmable, e.g. Crypto, vRAN-specific adapter).                               
+Firmware-programmable adapter                    compute-fpga            ❌                           ✅                                      Labels a host/node that includes a consumable Firmware-programmable adapter (programmable, e.g. Network/storage FPGA with programmable part of firmware image).
+SmartNIC enabled                                 network-smartnic        ❌                           ✅                                      Labels a host/node that includes a Programmable accelerator for vSwitch/vRouter, Network Function and/or Hardware Infrastructure.                              
+SmartSwitch enabled                              network-smartswitch     ❌                           ✅                                      Labels a host/node that is connected to a Programmable Switch Fabric or TOR switch                                                                             
+================================================ ======================= =========================== ====================================== ================================================================================================================================================================= =============================================
+
+**Table 2-1:** Profile extensions
+
+   \*\ **Note:** This is an initial set of proposed profiles and profile extensions and it is expected that more profiles and/or profile extensions will be added as more requirements are gathered and as technology enhances and matures.
+
+.. |Figure 2-1: Edge CDN with eMBB Core Network Slicing.| image:: ../figures/Fig2-1-uc1.png
+.. |Figure 2-2: Edge Private 5G with Core Network Slicing| image:: ../figures/Fig2-2-uc2.png
+.. |Figure 2-3: Edge Automotive (V2X) with uRLLC Core Network Slicing.| image:: ../figures/Fig2-3-uc3.png
+.. |Figure 2-4: Edge vRAN Deployments.| image:: ../figures/Fig2-4-uc4.png
+.. |Figure 2-1: Infrastructure profiles proposed based on VNFs categorisation.| image:: ../figures/RM-ch02-node-profiles.png
```

Comment:
Adjust based on changes recommended above

Reply:
I've fixed this in e44f71e

Reply:
This was implicitly fixed by using inline images.